backend/worker.py

```diff
@@ -121,15 +37,14 @@ def remove_permissions(remove_hierarchy, institution):
             for child in institution.children_institutions:
                 child = child.get()
                 if(child.verify_connection(institution, 'PARENT')):
-                    remove_permissions(remove_hierarchy, child)
+                    remove_hierarchy_permissions(remove_hierarchy, child)
```

Comment:
I didn't get it. You've renamed the method?

Reply:
Sim, pq antes ele tinha outra função com o mesmo nome dentro do arquivo. Essa função no caso remove as permissões hierarquicamente também.